source/iterators.tex

```diff
@@ -1099,18 +1099,14 @@
 
 \begin{itemdescr}
 \pnum
-\requires
+\expects
 \tcode{n}
-shall be negative only for bidirectional and random access iterators.
+shall be negative only for bidirectional iterators.
```

Comment:
Do not use "shall" within *Effects*: elements. We're describing a condition, not a requirement.

Reply:
Fixed.